ansible/roles/captain/tasks/main.yml

```diff
@@ -56,6 +56,14 @@
   tags:
     - localsettings
 
+- name: copy fab config
+  sudo_user: "{{ cchq_user }}"
+  template:
+    src: config.py.j2
+    dest: "{{ captain_src_dir }}/submodules/commcare-hq-deploy/fab/config.py"
+  tags:
+    - localsettings
```

Comment:
just easier to deploy captain. it runs this and captain's localsettings. this won't get run on a normal localsettings update since `deploy_captain` playbook is not included